src/dss_cell.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+#pragma once
+
+#include <vector>
+
+#include <common_types.hpp>
+#include <util/debug.hpp>
+
+namespace nest {
+namespace mc {
+
+/// data spike source: A cell that generates spikes provided as a vector of
+/// floating point valued spiketimes at the start of a run
+
+class dss_cell {
+public:
+    struct dss_cell_description {
```

Comment:
With the refactor asked later in the PR I had to move it outside. For this specific cell the discussion is moot.